backend/data/src/main/kotlin/io/tolgee/component/machineTranslation/providers/DeeplApiService.kt

```diff
@@ -34,7 +34,9 @@ class DeeplApiService(
     requestBody.add("text", text)
     requestBody.add("source_lang", sourceTag.uppercase())
     requestBody.add("target_lang", targetTag.uppercase())
-
+    deeplMachineTranslationProperties.optionalParameters?.map {
```

Comment:
since we don't need to return anything we should go for`forEach` rather then map.